revenuecat_examples/purchase_tester/lib/src/app.dart

```diff
@@ -55,6 +55,8 @@ class _MyAppState extends State<InitialScreen> {
       }
     });
 
+    Purchases.showStoreMessages(types: {InAppMessageType.billingIssue});
```

Comment:
Also need to remove this. Maybe we can add a button somewhere to call this if needed.